tests/roles/dataplane_adoption/tasks/nova_verify.yaml

```diff
@@ -9,8 +9,8 @@
   when: prelaunch_test_instance|bool
   ansible.builtin.shell: |
     {{ nova_header }}
-    ${BASH_ALIASES[openstack]} server list | grep -qF '| test | ACTIVE |' && ${BASH_ALIASES[openstack]} server stop test || echo PASS
-    ${BASH_ALIASES[openstack]} server list | grep -qF '| test | SHUTOFF |' || echo FAIL
+    ${BASH_ALIASES[openstack]} server list -c Name -c Status -f value | grep -qF "test ACTIVE" && ${BASH_ALIASES[openstack]} server stop test || echo PASS
```

Comment:
As you're changing it in the tests, please also update this in docs - both must be identical

Reply:
also make sure that the exit codes handling is not changing after switching the grep to cli args

Reply:
Updated the docs.
I've switched back to using grep so exit codes are consistent with old behavior.

* test VM exists
$ openstack server list -c Name -c Status -f value | grep -qF "test ACTIVE"
$ echo $?
0

* no-test doesn't exist
$ openstack server list -c Name -c Status -f value | grep -qF "no-test ACTIVE"
$ echo $?
1